### PR TITLE
Add documentation for redisquota adapter in rate limiting doc

### PR DESCRIPTION
**Pull request by gargnupur**

### CODE REVIEW

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 1]
```diff
@@ -55,13 +55,113 @@ so the configuration to enable rate limiting on both adapters is the same.
         * `memquota adapter` defines memquota adapter configuration.
         * `quota rule` defines when quota instance is dispatched to the memquota adapter.
 
-    Run the following command to enable rate limits.
+    Run the following command to enable rate limits using memquota.
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Replace . with :

File: .spelling [hunk 2]
```diff
@@ -72,6 +72,7 @@ Exfiltrating
 ExternalName
 Firebase
 FitStation
+FIXED_WINDOW
```
Comment (geeknoid) [inline .spelling]: Shouldn't be in the dictionary, put backticks around use instead.

File: .spelling [hunk 3]
```diff
@@ -142,6 +143,7 @@ Redis-based
 Registrator
 Reviewer1
 Reviewer2
+ROLLING_WINDOW
```
Comment (geeknoid) [inline .spelling]: Similar, shouldn't be in dict.

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 4]
```diff
@@ -55,13 +55,113 @@ so the configuration to enable rate limiting on both adapters is the same.
...
     $ kubectl apply -f @samples/bookinfo/policy/mixer-rule-productpage-ratelimit.yaml@
     {{< /text >}}
 
-1. Confirm the `memquota` handler was created:
+    Or
+
+    Save the following yaml file as `redisquota.yaml`. Replace [rate_limit_algorithm](/docs/reference/config/policy-and-telemetry/adapters/redisquota/#Params-QuotaAlgorithm),
+    [redis_server_url](/docs/reference/config/policy-and-telemetry/adapters/redisquota/#Params) with values for your configuration.
+
+    {{< text yaml >}}
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Does this yaml content exist on GitHub? If so, we should just link to that instead.

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 5]
```diff
@@ -95,6 +195,50 @@ so the configuration to enable rate limiting on both adapters is the same.
...
+    if no overrides match, is `500` requests per one second (`1s`). It is using ROLLING_WINDOW
+    algorithm for quota check and thus define `bucketDuration` of 500ms for ROLLING_WINDOW
+    algorithm. Three overrides are also defined:
+
+    * The first is `1` request (the `maxAmount` field), if the `destination` is `reviews`.
+    * The second is `500`, if the destination is `productpage` and source
+      is `10.28.11.20`
+    * The third is `2`, if the `destination` is `productpage`.
+
+    When a request is sent to the first matching override is picked (reading from top to bottom).
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/rate-limiting/index.md]: When a request is processed, the first

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 6]
```diff
@@ -111,7 +255,7 @@ so the configuration to enable rate limiting on both adapters is the same.
         destinationVersion: destination.labels["version"] | "unknown"
     {{< /text >}}
 
-    The `quota` template defines three dimensions that are used by `memquota`
+    The `quota` template defines three dimensions that are used by `memquota\redisquota`
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/rate-limiting/index.md]: by memquota or redisquota

Comment (geeknoid) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Here and elsewhere

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 7]
```diff
@@ -239,9 +383,12 @@ returns status `HTTP 429` to the caller.
 The `memquota` adapter uses a sliding window of sub-second resolution to
 enforce rate limits.
 
+The `redisquota` adapter can be configured to use either [ROLLING_WINDOW or FIXED_WINDOW](docs/reference/config/policy-and-telemetry/adapters/redisquota/#Params-QuotaAlgorithm)
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/rate-limiting/index.md]: either the

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 8]
```diff
@@ -252,12 +399,20 @@ namespace.
 
 ## Cleanup
 
-1. Remove the rate limit configuration:
+1. Remove the `memquota` rate limit configuration, if using `memquota`:
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Is using memquota, remove the...

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 9]
```diff
@@ -252,12 +399,20 @@ namespace.
...
 
     {{< text bash >}}
     $ kubectl delete -f @samples/bookinfo/policy/mixer-rule-ratings-ratelimit.yaml@
     {{< /text >}}
 
+    Or
+
+    Remove the `redisquota` rate limit configuration, if using `redisquota`:
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/rate-limiting/index.md]: If using redisquota, remove

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 10]
```diff
@@ -252,12 +399,20 @@ namespace.
...
+
+    {{< text bash >}}
+    $ kubectl delete -f @redisquota.yaml@
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Is this is a GitHub file, you need to put the full path within the @@ (relative to the root of the repo)
If this is not a GitHub file, remove the @@

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md (hunk 1 above)
Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: FIxed

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md (hunk 4 above)
Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: No, it doesn't... and guess it's too late to add to istio 1.0 now..

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md (hunk 5 above)
Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Fixed

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md (hunk 6 above)
Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Fixed

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md (hunk 7 above)
Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Fixed

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md (hunk 9 above)
Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Fixed

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md (hunk 10 above)
Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Fixed

File: content/docs/tasks/policy-enforcement/rate-limiting/index.md (hunk 8 above)
Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/rate-limiting/index.md]: Fixed

File: .spelling (hunk 2 above)
Comment (gargnupur) [reply .spelling]: Fixed

File: .spelling (hunk 3 above)
Comment (gargnupur) [reply .spelling]: Fixed

**Pull request merged**